ERCS/erc-6860.md

```diff
@@ -293,8 +304,18 @@ contractName    = address
                 / domainName
 chainid         = %x31-39 *DIGIT
 
-pathQuery       = mPathQuery ; path+query for manual mode
-                / aPathQuery ; path+query for auto mode
+pathQuery       = mPathQuery [ evmQuery ] ; path+query for manual mode
+                / aPathQuery [ evmQuery ] ; path+query for auto mode
+
+evmQuery        = *[(evmPath evmParams)]
+evmPath         =  "?" ; begins with "/" but not "//" 
```

Comment:
This seems to allow a wired format like

`web3://usdc.eth/balanceOf/vitalik.eth?returns=(uint256)?_blockNumber=1234`?

I think the problem here is that these parameters `_xxxxx=xxx` should be part of `mPathQuery` and `aPathQuery` while appending `evmQuery` may introduce unexpected formats.